mlir/python/mlir/dialects/affine.py

```diff
@@ -0,0 +1 @@
+from ._affine_ops_gen import *
```

Comment:
Newline please.

Reply:
Fixed and took the opportunity to add the copyright template into it as well.  Thanks!